src/routers/asset.ts

```diff
@@ -227,54 +226,6 @@ export function handle(context: IndexerContext, router: Router) {
         }
     );
 
-    /**
-     * @swagger
-     * /asset-image/{assetType}:
-     *   get:
-     *     summary: Returns asset image of the specific assetType
-     *     tags: [Asset]
-     *     parameters:
-     *       - name: assetType
-     *         description: The type of the Asset
-     *         required: true
-     *         in: path
-     *         type: string
-     *     responses:
-     *       200:
-     *         description: asset image
-     *         schema:
-     *           type: image
-     */
-    router.get(
-        "/asset-image/:assetType",
-        validate({
-            params: {
-                assetType: assetTypeSchema.required()
-            }
-        }),
-        async (req, res, next) => {
-            const assetTypeString = req.params.assetType;
-            try {
-                const assetType = new H160(assetTypeString);
-                const assetImageInst = await AssetImageModel.getByAssetType(
-                    assetType
-                );
-                if (!assetImageInst) {
-                    res.status(404).send("Not found");
-                    return;
-                }
-                const { image } = assetImageInst.get();
```

Comment:
The type of image is Buffer. You can read an image file from a file and use it.

https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options
https://stackoverflow.com/a/46441727